Sources/App/Models/Creator/Creator.swift

```diff
@@ -46,13 +51,38 @@ final class Creator: Model {
   // sourcery:end
 }
 
+extension Creator: JSONRepresentable {
+
+  func makeJSON() throws -> JSON {
```

Comment:
Why not generate?

Reply:
How to say sourcery to do this: 
```swift
try json.set(Keys.photoUrl, photoURL())
```
??

Reply:
Easily :) we're supporting it 
add  `// sourcery: nestedJSONField` above `func photoURL()`
and to ignore appearing this property field in JSON add `// sourcery: ignoreInJSON` above `  var photoUrl: String`

It's described in Wiki for sourcery)

Reply:
Check it

Reply:
hm, I'm check it, problem is here:
```Swift
...
struct Keys {
    static let id = "id"
    static let userId = "user_id"
    static let position = "position"
    static let photoUrl = "photo_url"
    static let info = "info"
    static let url = "url"
    static let active = "active"
    static let photoURL = "photo_url"
  }
```
double  `static let photoURL = "photo_url"` there was no have time to resolve it, what is why I didn't use toJSON generation

Reply:
fixed, now we have photoURL() as warp on nested user.photoURL.